01_login.php

```diff
@@ -54,134 +55,132 @@
 
 		<!-- Navbar (sit on top) -->
 		<div class="w3-top">
-			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
+			<div class="w3-bar w3-grey w3-padding w3-card">
```

Comment:
added colour within the navbar: grey to fit the colour scheme also removed the wide effect as i did not like the aesthetic of it

Reply:
used w3-schools for this technique - was easier to adjust and personalize